### PR TITLE
Print more informantion about cached Docker images linux

### PR DESCRIPTION
**Pull request by dsame**
# Description
Improvement

Add too docker information listed in Software report the docker digest and age. 

## Check list
- [x] Related issue / work item is attached
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by maxim-lobanov — APPROVED** [review]

**Comment by AlenaSviridenko** [comment]
@dsame this output format looks unclear
image


I'd suggest to create a table like the one below
| Repository:Tag | Digest | Created |
| - | - | - |
|node:12| sha256:b7aa57bb141b7adca179797a88f6dc350603124f9a88e4d78d9a67d0ad129350 | 13 days ago |

**Review by miketimofeev — APPROVED** [review]

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 1]
```diff
@@ -240,6 +240,17 @@ function Get-CachedDockerImages {
     return $images
 }
 
+function Get-CachedDockerImagesTableData {
```
Comment (al-cheb) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:243]: Why don't you use json format?

```
docker images --digests --format='{{json .}}' | ConvertFrom-Json | Select-Object @{n="Repository:Tag";e={"$($_.Repository):$($_.Tag)"}},Digest,@{n="Created" ; e = {$_.CreatedAt.Split()[0]}}

Repository:Tag Digest                                                                  Created
-------------- ------                                                                  -------
ubuntu:latest  sha256:fff16eea1a8ae92867721d90c59a75652ea66d29c05294e6e2f898704bdb8cf1 2020-10-23
alpine:latest  sha256:185518070891758909c9f839cf4ca393ee977ac378609f700f60a771a2dfe321 2020-05-30
```

File: images/macos/provision/core/pipx-packages.sh [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#!/bin/bash -e -o pipefail
```
Comment (al-cheb) [inline images/macos/provision/core/pipx-packages.sh:1]: Why do we need to change the pipx-packages.sh script  in that PR?

File: images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1 (hunk 1 above)
Comment (dsame) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1:243]: It is inherited code, i guess for sake of normalisation: stripping headers and having named fields. It is looking nicer in powershell context indeed over sh-like transformations

**Pull request merged**